src/sqltest/parts/commandLine/commandLineService.test.ts

```diff
@@ -2,6 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+/* Disabled pending next vscode merge which allows electron module to be imported during test runs
```

Comment:
can you open an issue to reenable these tests post-merge?

Reply:
https://github.com/Microsoft/azuredatastudio/issues/4060